includes/CLI/Plugin_Check_Command.php

```diff
@@ -266,18 +277,25 @@ public function list_checks( $args, $assoc_args ) {
 
 		$all_checks = array();
 
-		foreach ( $check_repo->get_checks() as $key => $check ) {
+		foreach ( $check_repo->get_checks( Default_Check_Repository::TYPE_ALL | Default_Check_Repository::INCLUDE_EXPERIMENTAL ) as $key => $check ) {
```

Comment:
These constants or on the `Check_Repository` class, not `Default_Check_Repository`.

Also, can we do the filtering here instead? The checks collection returned by `get_checks()` has a `filter` method for this purpose.

```suggestion
		foreach ( $check_repo->get_checks( Check_Repository::TYPE_ALL | Check_Repository::INCLUDE_EXPERIMENTAL )->filter(
			static function ( Check $check ) {
				// Filter by $check->get_categories() and $check->get_stability()
				// return bool
			}
		) as $key => $check ) {
```

Reply:
I think retrieving all checks here is not the right approach. IMO it should behave the same way as `wp plugin check` does:
* By default, only stable checks are returned.
* Instead of a `--stability` option, we should use a boolean flag option `--include-experimental`.
* Only if that flag is set, we should get both stable and experimental checks.